cloud/aws/modules/ecs_fargate_service/variables.tf

```diff
@@ -0,0 +1,134 @@
+#------------------------------------------------------------------------------
+# Misc
+#------------------------------------------------------------------------------
+variable "app_prefix" {
+  description = "App prefix for naming resources on AWS. For most resources, -civiform is added to this string."
+}
+
+#------------------------------------------------------------------------------
+# AWS Networking
+#------------------------------------------------------------------------------
+variable "vpc_id" {
+  description = "ID of the VPC"
+}
+
+#------------------------------------------------------------------------------
+# AWS ECS SERVICE
+#------------------------------------------------------------------------------
+variable "ecs_cluster_arn" {
+  description = "ARN of an ECS cluster"
+}
+
+variable "desired_count" {
+  description = "(Optional) The number of instances of the task definition to place and keep running. Defaults to 0."
+  type        = number
+  default     = 1
```

Comment:
**suggestion (blocking):** 
```suggestion
  description = "(Optional) The number of instances of the task definition to place and keep running. Defaults to 2."
  type        = number
  default     = 2
```

Reply:
> Do you know if we have any existing deployments that set this to 2? Wondering if that's ever been tested.

Good question. I'd thought that all deployments had at least two server nodes running so this is coming as a surprise. I know Seattle does with their old config.

Reply:
I'm ok merging this as-is if this is at parity with the current system, but definitely something to improve soon to ensure server redundancy in prod envs